test/settings_test.dart

```diff
@@ -436,8 +448,11 @@ void main() {
 
     group('Feedback form', () {
```

Comment:
Add a test that you can open the sources page.

Reply:
@AdrianMargineanu  can help with this?